classes/quiz.class.php

```diff
@@ -0,0 +1,168 @@
+<?php
+// This file is part of Moodle - http://moodle.org/
+//
+// Moodle is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Moodle is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
+
+/**
+ * Helper class for plagiarism_turnitinsim component in quizzes.
+ *
+ * @package   plagiarism_turnitinsim
+ * @copyright 2020 Turnitin
+ * @author    David Winn <dwinn@turnitin.com>
+ * @license   http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
+ */
+
+defined('MOODLE_INTERNAL') || die();
+
+/**
+ * Helper class for plagiarism_turnitinsim component in workshops
```

Comment:
This will sound silly... but Does the checkstyle fail if the comment doesn't end in a period?

Reply:
I think it does, yes. Good eyes!

Reply:
I think that's only for single line comments. Bigger mistake here is not changing the word "workshops" to "quiz". I'll run it through the checkstyle before I finish to double check.